### PR TITLE
Integrating Leaflet-Environmental-Library .

### PR DESCRIPTION
**Pull request by sagarpreet-chadha**
#268 

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] tests pass -- `rake test`
* [x] code has been rebased on top of latest master (check if another pull request was added recently, and please rebase)
* [x] pull request is descriptively named and, if possible, multiple commits squashed if they're smaller changes

Thanks!

### CODE REVIEW

**Comment by sagarpreet-chadha** [comment]
How does this look ?

screen shot 2018-07-07 at 1 08 44 pm

**Comment by sagarpreet-chadha** [comment]
@jywarren ...in the map there are 2 grey lines  (see pic above) . What could be the possible reason ?

**Comment by sagarpreet-chadha** [comment]
Hi @jywarren ...can you confirm this by running on localhost ?Thanks !

**Comment by sagarpreet-chadha** [comment]
The grey lines appear even if i remove the **leaflet-environmental-layers** completely .

**Comment by sagarpreet-chadha** [comment]
Okay Finally found --- its because of Bootstrap's CSS that adds those grey spaces in leaflet map .

**Comment by sagarpreet-chadha** [comment]
Hi ...there seems to be no way of disabling the inheritance of CSS property .
One solution is to use CSS grid instead of Bootstrap .
What should we do ?

**Comment by jywarren** [comment]
Oh, that's unfortunate. Can you link to anyone else having this issue, or docs? I feel we should be able to override the CSS... do you know what rule in particular is causing it?

**Comment by sagarpreet-chadha** [comment]
This is the same issue : 
https://gis.stackexchange.com/questions/112100/bootstrap-and-leaflet-grid-issue

And this is related issue : 
https://github.com/Leaflet/Leaflet/issues/3862

I am not able to find which CSS in bootstrap is causing this .

**Comment by jywarren** [comment]
Hi @sagarpreet-chadha sorry i couldn't work on this earlier, but i like your solution of just putting it full-width above the grid. Let's do that!

**Comment by sagarpreet-chadha** [comment]
Done @jywarren ...kindly review this 👍 .

**Comment by jywarren** [comment]
Could you screenshot? Thank you!!!

**Comment by sagarpreet-chadha** [comment]
Screenshot : 

screen shot 2018-07-08 at 2 45 45 pm

**Comment by sagarpreet-chadha** [comment]
And reminder : to run `bower install` 😄 .

**Comment by jywarren** [comment]
Hi! Do you think we could make it completely full width - like 100% of the
window?

Looking great! I'm also wondering about the markers, when they overlap a
lot they kind of get hard to see. Maybe we should try to use the default
Leaflet ones, but see if we can make them smaller, or recolor them red?
Just an idea. But would that happen upstream in the library itself?

**Comment by sagarpreet-chadha** [comment]
HI !
Yes the change in marker will be in the library itself ...we can do that 👍  and then you can do a version bump ?

**Comment by sagarpreet-chadha** [comment]
screen shot 2018-07-12 at 11 43 24 pm

How does this look ? 
If it is good to go we can merge this after we finalize the marker icon here in this PR https://github.com/publiclab/leaflet-environmental-layers/pull/31 .

File: bower.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "glfx-js": "git+https://github.com/jywarren/glfx.js.git",
     "webgl-distort": "jywarren/webgl-distort",
     "image-sequencer": "git+https://github.com/publiclab/image-sequencer.git#~1.4.0",
-    "junction": "theleagueof/junction"
+    "junction": "theleagueof/junction",
+    "leaflet-environmental-layers": "*"
```
Comment (sagarpreet-chadha) [inline bower.json]: This will take latest version always , right ? 
Can you confirm this @jywarren  ?

**Comment by jywarren** [comment]
I think we can merge this and then just not yet deploy to MK until we're ready in the upstream library. Going ahead! Thanks!!!

**Pull request merged**